monkey/tests/unit_tests/infection_monkey/test_monkey.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+import pytest
+from tests.data_for_tests.otp import OTP
+
+from infection_monkey.model import AGENT_OTP_ENVIRONMENT_VARIABLE, OTP_FLAG
+from infection_monkey.monkey import InfectionMonkey
+
+
+@pytest.fixture(autouse=True)
+def configure_environment_variables(monkeypatch):
+    monkeypatch.setenv(AGENT_OTP_ENVIRONMENT_VARIABLE, OTP)
+    monkeypatch.setenv(OTP_FLAG, True)
+
+
+def test_get_otp(monkeypatch):
+    assert InfectionMonkey._get_otp() == OTP
+    assert AGENT_OTP_ENVIRONMENT_VARIABLE not in os.environ
+
+
+def test_get_otp__no_otp(monkeypatch):
+    monkeypatch.delenv(AGENT_OTP_ENVIRONMENT_VARIABLE)
+    with pytest.raises(Exception):
+        InfectionMonkey._get_otp()
+
+
+def test_get_otp__feature_flag_disabled(monkeypatch):
```

Comment:
We want to unit test feature flags? Interesting... We should come up with a feature flag naming convention, as `FLAG` is a term often used in cmd arguments and lower level code. Maybe `F_FLAG`? So it would be `OTP_F_FLAG`?

Reply:
It was easy enough to do it and it helped me quickly test that the feature flag was working as expected.